pdp/portals/bc_prism.py

```diff
@@ -2,44 +2,85 @@
 the 1971-2000 and 1981-2010 climatologies and monthly climate data for
 800 meter resolution PRISM dataset for BC
 '''
-
 from pdp.portals import make_raster_frontend, data_server
 from pdp_util.ensemble_members import EnsembleMemberLister
-import re
 
 
 __all__ = ['url_base', 'mk_frontend', 'mk_backend']
 
 
-ensemble_name = 'bc_prism_with_monthlies'
+ensemble_name = 'bc_prism_monthly_and_climos'
 url_base = '/bc_prism'
 title = 'High-Resolution PRISM Data'
 
 
-class PrismEnsembleLister(EnsembleMemberLister):
-    def describe_time_set(self, unique_id):
-        monthly = unique_id.find("mon") != -1
-        climatology = unique_id.find("Clim") != -1
-        date_finder = re.match(r'.*_(\d+)-(\d+).*', unique_id)
+def resolution_name(time_resolution):
+    return time_resolution.capitalize()
 
-        return "{} {} {}-{}".format("Monthly " if monthly else "",
-                                    "Climatological Averages " if climatology
-                                    else "Timeseries ",
-                                    date_finder.group(1)[0:4],
-                                    date_finder.group(2)[0:4])
 
+class PrismEnsembleLister(EnsembleMemberLister):
+    
     def list_stuff(self, ensemble):
+        """
```

Comment:
Excellent work on both the docstring update and the clearer description of the datasets for the portal menu.